### ISSUE TITLE
[SIMD]: Use Java Vector API for SIMD operations

### ISSUE THREAD

**Issue by mfirry** [dylibso/chicory#695]
This issue is mainly for me to keep track of the current _porting_ of SIMD-related operations to leverage on Java Vector API.
I'll fork, branch out and little by little push my work. Suggestions, advices and ideas are more than welcome.

- [x] I8x16_EQ
- [x] I8x16_ADD
- [x] I8x16_SUB
- [x] I8x16_SHL
- [x] I8x16_ALL_TRUE
- [x] F32x4_MUL
- [x] F32x4_ABS
- [x] F32x4_MIN
- [x] V128_NOT
- [x] V128_BITSELECT
- [x] I8x16_EXTRACT_LANE_S
- [ ] F32x4_CONVERT_I32x4_U
- [ ] I32x4_TRUNC_SAT_F32x4_S
- [ ] I8x16_SWIZZLE

**Comment by andreaTP** [comment]
Yes @wendigo , I'm planning to release a 1.1 by the end of the week.
Please consider that Simd support is only in the interpreter at the moment.

**Comment by andreaTP** [comment]
released in `1.1.0`
